curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md

```diff
@@ -28,6 +28,8 @@ tests:
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(MyToDoList)); return mockedComponent.find('MyToDoList').length === 1; })());
   - text: The first child of <code>MyToDoList</code> should be a <code>textarea</code> element.
     testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(MyToDoList)); return mockedComponent.find('MyToDoList').children().childAt(0).type() === 'textarea'; })());
+    text: The second child of <code>MyToDoList</code> should be a <code>br</code> element.
```

Comment:
```suggestion
  - text: The second child of <code>MyToDoList</code> should be a <code>br</code> element.
```
Sorry, missed this when reviewed before.

Reply:
@ojeytonwilliams sry I missed that

Reply:
No worries, that's what reviews are for.